src/fragments/MyChannels.tsx

```diff
@@ -52,11 +60,46 @@ const MyChannels: React.FC = () => {
     });
   }, [currentUser]);
 
+  useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const test: any = currentNotification?.data.message;
```

Comment:
The SDK provides type helper functions to prevent unsafe type casting.

`isSystemSentMessageNotification`
`isUserSentMessageNotification`
`isUserRepliedToMessageNotification`
`isUserMentionedNotification`
`isUserMentionedChannelNotification`